### DISCUSSION TITLE
HTML on title

### DISCUSSION THREAD

**Discussion by lenineto** [ditdot-dev/vue-flow-form#261] [Q&A]
Hi guys,

I'm wondering if there is a way to pass html tags inside a question's title. 
I'm using the <question> tags rather than QuestionModel, so I can pass answers to another questions, but I need to add line breaks in some titles.

**Comment by spinn** [comment]
Questions as components don't support HTML unfortunately. We'd need to create slots for this, which is possible and which we could look to maybe implement in a future version.
